scenarios/kubernetes_build.py

```diff
@@ -31,6 +32,36 @@ def check(*cmd):
     subprocess.check_call(cmd)
 
 
+def get_gcs_path(args):
+    # xref https://github.com/kubernetes/release/blob/master/push-build.sh
```

Comment:
xref seems gross

Any chance we can get push-build.sh to write this file, since this is what controls the behavior?